### ISSUE TITLE
Default REST doesn't work

### ISSUE THREAD

**Issue by RSalo** [tuyakhov/yii2-json-api#21]
I try to use default yii rest, but model doesn't save anything.

I did request with ember to POST http://localhost/v1/users with data
http://image.prntscr.com/image/b450d336c78e4dfdb26574d55e5784b3.png

After some investigations, i found the reason.
https://github.com/tuyakhov/yii2-json-api/blob/master/src/JsonApiParser.php#L33 make data format like this:
```
array(1) {
  ["User"]=>
  array(2) {
    ["email"]=>
    string(5) "eeeee"
    ["password"]=>
    string(4) "cccc"
  }
}
```
in yii\rest\CreateAction https://github.com/yiisoft/yii2/blob/master/framework/rest/CreateAction.php#L51 load wrong set attributes that are empy after setAttributes and $model->save() can't save it and always return validator error
```
    "errors": [
        {
            "source": {
                "pointer": "/data/attributes/email"
            },
            "detail": "Email cannot be blank."
        },
        {
            "source": {
                "pointer": "/data/attributes/password"
            },
            "detail": "Password cannot be blank."
        }
    ]
}
```

**Comment by tuyakhov** [comment]
@RSalo Yes, this is a known issue. That has been discussed earlier (https://github.com/tuyakhov/yii2-json-api/issues/20#issuecomment-281619005)
The default REST actions of the Yii2 is not compatible with JsonApi specification. I will add create/update/delete actions very soon. Stay tuned it will be available in next release.
Thank you for reporting.

**Comment by tuyakhov** [comment]
The Json api compatible actions have been implemented in `v0.1.1`
